modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/Amount.scala

```diff
@@ -1,31 +1,59 @@
 package org.tessellation.rosetta.domain
 
+import cats.implicits.catsSyntaxEitherId
```

Comment:
we tend to import needed cats implicits like this:
`import cats.syntax.either._`

Reply:
@TheMMaciek Yes; I use IntelliJ to "quick fix" missing imports and I forgetting I have to fix the Cats implicits manually. I haven't found a way to configure IntelliJ to use a different import -- do you know how to do that?

Reply:
I don't actually. I just check it and change manually.